apis/hoh_heartbeat/v1/hoh_heartbeat_types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// HohHeartbeatSpec defines the desired state of HohHeartbeat
+type HohHeartbeatSpec struct{}
```

Comment:
I think you can just remove the spec field alltogether. Kubernetes CRs do not have to have spec.

Reply:
done